wiki/Tournaments/OWC/2013/en.md

```diff
@@ -279,6 +280,8 @@ Mappool
 - Tiebreaker
   - **[DJ Okawari - Luv Letter (nold\_1702) \[Posthumous\]](https://osu.ppy.sh/beatmaps/127363)**
 
+--------------------------------
+
 ## Match Results
 
 ### Finals
```

Comment:
Formatting for results moves the flags towards the center, so the team B (on the right should start with the flag and then the team name, an example:
`| **Korea** ![][flag_KR] | **6** | 5 | Taiwan ![][flag_TW] |`
should be
 `| **Korea** ![][flag_KR] | **6** | 5 | ![][flag_TW] Taiwan |`
Repeat on every table from here onwards.